Mlem/API/Requests/ResolveObject.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ResolveObject.swift
+//  Mlem
+//
+//  Created by Jonathan de Jong on 14.06.2023.
+//
+
+import Foundation
+
+// lemmy_api_common::site::ResolveObject
+struct ResolveObjectRequest: APIGetRequest {
+
+    typealias Response = ResolveObjectResponse
+
+    let instanceURL: URL
+    let path = "resolve_object"
+    let queryItems: [URLQueryItem]
+
+    init(
+        account: SavedAccount,
+
+        query: String
+    ) {
+        self.instanceURL = account.instanceLink
+        self.queryItems = [
+            .init(name: "q", value: query),
+
```

Comment:
would you object to removing these blank lines here and in the others?

in a request the size of `EditSite` I think the occasional break works to help split out the sections, but in many of the others like here it just looks off?

I'm thinking it's designed to separate the auth/account argument from the other fields, but with so few it doesn't feel necessary?

Reply:
> would you object to removing these blank lines here and in the others?

they're mainly meant for me to easily insert and discern the properties that're not `auth`, I think it works for code readability like that.

Reply:
no worries, personally I'd always put the auth argument first, but as developers we can debate coding style until the heat death of the universe and never reach consensus 🙃 

On a related note, I was planning to bring [swiftformat](https://github.com/nicklockwood/SwiftFormat) into the project when I get a moment, likely set up as a pre-commit hook, what are your thoughts?

The thinking is it'll ensure the project is formatted to the _"official"_ language style, and saves any style based stuff creeping into reviews?

Reply:
I'll make an issue about this, so we can introduce it later